tests/contributors.sh

```diff
@@ -2,18 +2,18 @@
 # Copy the latest contributors.md file content to afterword
 # this will publish the contributors file online in The Turing Way book
 all_contributors='../../README.md' # Path to the main README file of The Turing Way
-contributor_highlights='../../contributors.md'
-contributor_record='afterword/contributors-record.md'
+contributors_highlight='../../contributors.md'
+contributors_record='../book/website/afterword/contributors-record.md'
```

Comment:
I wonder if these lines should be something like
```
contributors_highlight=$(find . -type f -name contributors.md)
```
so that they can be reliably found in the GHA file structure?

Reply:
This makes sense - as long as there are no other files with this name. 
Can you edit the file accordingly?